stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetActivity.kt

```diff
@@ -72,7 +72,10 @@ internal class PaymentSheetActivity : BasePaymentSheetActivity<PaymentResult>()
             finish()
             return
         }
-        val fragmentArgs = bundleOf(EXTRA_STARTER_ARGS to starterArgs)
+
+        viewModel.fetchIsGooglePayReady()
+        viewModel.updatePaymentMethods()
+        viewModel.fetchPaymentIntent()
```

Comment:
Looks like this will make a network call every time the fragment is created. It might be worth saving the state rather than fetching every time.